filebeat/values.yaml

```diff
@@ -83,7 +87,7 @@ deployment:
   labels: {}
   affinity: {}
   # Include the deployment
-  enabled: false
+  enabled: true
```

Comment:
this doesn't seem related to the `extraArgs` change

Reply:
fixed

Reply:
I'm still seeing this param being changed from false to true in this PR.